pages/create/index.tsx

```diff
@@ -20,6 +30,64 @@ export default function Create() {
     await createUser(userData)
   }
 
+  const handleCreateEvent = async () => {
+    const eventData: EventData = {
+      title: 'Event title',
+      date: new Timestamp(2000000, 0),
+      foodbankId: '',
+      foodAmount: 0,
+      location: new GeoPoint(10, 10),
+      groupId: undefined,
+    }
+
+    await createEvent(eventData)
+  }
+
+  const handleCreateGroup = async () => {
+    const allUsers = await getAllUsers()
+
+    if (allUsers.length < 5) return
+
+    const groupMembers = [allUsers[1].id, allUsers[2].id, allUsers[3].id]
+
```

Comment:
Why did you use the ceiling and floor functions here? It could just be replaced with:
`const min = 0`
`const max = 5`